component/nav/nav.css

```diff
@@ -0,0 +1,50 @@
+.nav {
+    /* width: 700px; */
```

Comment:
It's best to get rid of commented out code like this. There's always version control and git history if you want to bring something back :)